sktime/registry/_lookup.py

```diff
@@ -77,6 +78,12 @@ def all_estimators(
     as_dataframe: bool, optional (default=False)
                 if False, return is as described below;
                 if True, return is converted into a DataFrame for pretty display
+    return_tags: str or list of str, optional (default=None)
```

Comment:
the docstring should say more clearly what happens in the two cases `as_dataframe=True` and `as_dataframe=False`.

Reply:
Hey, I fleshed out the as_dataframe section of the docstring in more depth.  Let me know if the improvements are satisfactory.

Reply:
yes, excellent! I was wondering how to make the different interacting conditions clear here, introducing the "attributes returned" makes it a lot clearer. Return section is great.

Minor thing: instead of pandas dataframe, write `pandas.DataFrame`

Reply:
In the process of fixing these/style issues now!  Will add a new comment when I am ready for review!

Reply:
Thanks!  Ready for review, though no rush :)